library/encounter.inc.php

```diff
@@ -32,3 +32,10 @@ function fetchDateService($encounter)
     $result = explode(" ", $result['date']);
     return $result[0];
 }
+
+function fetchPosCode($facility_id)
+{
```

Comment:
maybe putting this function in the encounter service will be better long term since we've been moving away from the old libraries

Reply:
When does the engineering OpenEMR meeting occur? I would like to sit in on them when I can.

Reply:
sat afternoons 2 pm eastern :laughing: